nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/repository/FileSystemRepository.java

```diff
@@ -953,7 +941,7 @@ public synchronized void close() throws IOException {
                 // is called. In this case, we don't have to actually close the file stream. Instead, we
                 // can just add it onto the queue and continue to use it for the next content claim.
                 final long resourceClaimLength = scc.getOffset() + scc.getLength();
-                if (recycle && resourceClaimLength < maxAppendClaimLength) {
+                if (recycle && resourceClaimLength < MAX_APPENDABLE_CLAIM_LENGTH) {
                     // we do not have to synchronize on the writable claim queue here because we
                     // are only adding something to the queue. We must synchronize if we are
                     // using a ResourceClaim from the queue and incrementing the claimant count on that resource
```

Comment:
Another comment that longer applies.  Leaving it in might confuse our future selves.